django/contrib/admin/bin/compress.py

```diff
@@ -39,18 +39,18 @@ def main():
         if options.verbose:
             sys.stdout.write("No filenames given; defaulting to admin scripts\n")
         files = [
-            os.path.join(js_path, f) for f in
-            ["actions.js", "collapse.js", "inlines.js", "prepopulate.js"]
+            js_path / f
+            for f in ["actions.js", "collapse.js", "inlines.js", "prepopulate.js"]
         ]
     else:
-        files = options.file
-
-    for file_name in files:
-        if not file_name.endswith(".js"):
-            file_name = file_name + ".js"
-        to_compress = os.path.expanduser(file_name)
-        if os.path.exists(to_compress):
-            to_compress_min = "%s.min.js" % "".join(file_name.rsplit(".js"))
+        files = [Path(f) for f in options.file]
+
+    for file_path in files:
+        if not file_path.suffix == 'js':
```

Comment:
It would be nice to add a comment about that (`# Allow omitting the .js prefix in the script's arguments.`).